microprofile/WFLY-12681_Provide_a_microprofile-platform_Galleon_layer.adoc

```diff
@@ -0,0 +1,86 @@
+= WFLY-12681 Provide a 'microprofile-platform' Galleon layer
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Historically, we used to have a 'microprofile' layer spec aggregating all MicroProfile (further as MP) specs.
+At that moment it only contained config, health and metrics which got eventually renamed
+to 'observability' by https://issues.redhat.com/browse/WFLY-11774[WFLY-11774] (no RFE there).
+
+With the introduction of new the specs such as MP Fault Tolerance, MP OpenAPI, etc. we want to
+reintroduce the 'microprofile-platform' layer spec which contains all available MP specs in the server.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-12681[WFLY-12681 Provide a 'microprofile-platform' Galleon layer]
+* https://issues.redhat.com/browse/EAP7-1396[EAP7-1396 Provide a 'microprofile-platform' Galleon layer]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-11774[WFLY-11774 Rename microprofile layer to be observability]
+
+=== Dev Contacts
+
+* mailto:rhusar@redhat.com[Radoslav Husar]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* MicroProfile
+* Galleon
+
+=== Other Interested Projects
+
+N/A
+
+== Requirements
+
+=== Hard Requirements
+
+* New 'microprofile-platform' Galleon layer is available for provisioning.
+* Each MP spec is optional to allow for excluding during provisioning (this is a community-only feature).
+
+=== Nice-to-Have Requirements
+
+N/A
+
+=== Non-Requirements
+
+N/A
+
+== Implementation Plan
+
+No coordination required. Only thing that needs to make use of this are quickstart examples using bootable jars should
+make use of the new layer in the product.
+
+== Test Plan
+
+Testing is encompassed in the upstream testsuite in multiple places:
+
+* by the `testsuite/integration/basic` which provisions this layer and runs subset of MP tests from basic suite with it
```

Comment:
@rhusar I guess I already asked on the MR (so if you answer there I'll copy the answer here and vice-versa), anyway ... what criteria drove the selection/exclusion of tests?

Reply:
@tommaso-borgato OK, added a note there about this.